boards/nxp/mimxrt1060_evk/board.cmake

```diff
@@ -4,13 +4,19 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+if (NOT ("${BOARD_QUALIFIERS}" MATCHES "qspi"
```

Comment:
as above

Reply:
in this case there is a space, so I don't know what you meant here